packages/fxa-settings/src/pages/Legal/Privacy/index.test.tsx

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import LegalPrivacy, { viewName } from '.';
+import { screen, render, fireEvent, waitFor } from '@testing-library/react';
+import { usePageViewEvent, logViewEvent } from '../../../lib/metrics';
+import { FluentBundle } from '@fluent/bundle';
+import { getFtlBundle, testAllL10n } from 'fxa-react/lib/test-utils';
+import { REACT_ENTRYPOINT } from '../../../constants';
+import { fetchLegalMd } from '../../../lib/file-utils-legal';
+
+jest.mock('../../../lib/file-utils-legal');
+jest.mock('../../../lib/metrics', () => ({
+  usePageViewEvent: jest.fn(),
+  logViewEvent: jest.fn(),
+}));
+
+// There's not a good way to use react-markdown in tests until we use jest ESM. Using the jest
```

Comment:
@vbudhram This does make me wonder if we should have a functional test for these pages to check that the Markdown rendered. I can also file a small 2-pointer to investigate this and add better coverage; I didn't try enabling Jest ESM since it looked like it might be a few steps and might affect the whole test suite, but I did poke around and tried adding various recommendations I found on the web in a jest config instead with no luck. I tried downgrading these packages and there was a Typescript issue but that may work - it just didn't feel right pinning us to an old version of packages that help us parse external MD into HTML.

Reply:
@LZoog Yea I don't think it will hurt to do a functional test for this. You could probalby copy this example https://github.com/mozilla/fxa/blob/53977c4f7b198e5e0eff45f64f9c676f4f0ae7b1/packages/functional-tests/tests/react-conversion/cannotCreateAccount.spec.ts#L13

Reply:
Let's mention this to QA and/or file a 1-pointer.

Reply:
Ticket filed: https://mozilla-hub.atlassian.net/browse/FXA-6820